README.md

```diff
@@ -1,61 +1,59 @@
 # Screwdriver API
-[![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][wercker-image]][wercker-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
+[![Version][npm-image]][npm-url] [![Pulls][docker-pulls]](docker-url) ![Stars][docker-stars] [![Build Status][wercker-image]][wercker-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
 
-> API module for the Screwdriver CD service
+> API for the Screwdriver CD service
 
 ## Usage
 
+### From Source
+
 ```bash
-npm install screwdriver-api
+$ git clone git@github.com:screwdriver-cd/api.git ./
+$ npm install
+$ vim ./config/local.yaml # See below for configuration
+$ npm start
+info: Server running at http://localhost:8080
 ```
 
-```javascript
-const API = require('screwdriver-api');
-const Datastore = require('screwdriver-datastore-dynamodb');
-
-const server = new API({
-    datastore: new Datastore(),
-    port: 8666
-}, (error, server) {
-    if (error) {
-        return console.error(error);
-    }
-    console.log('Server running at ', server.info.uri);
-});
+### Prebuilt Docker image
+
+```bash
+$ vim ./local.yaml # See below for configuration
+$ docker run --rm -it --volume=`pwd`/local.yaml:/config/local.yaml -p 8080 screwdrivercd/api:latest
+info: Server running at http://localhost:8080
 ```
 
-It's also possible to override the config value passed to a resource plugin that is registered.
-The resource plugins being registered are:
-
-1. screwdriver-plugin-login
-2. screwdriver-plugin-builds
-3. screwdriver-plugin-jobs
-4. screwdriver-plugin-pipelines
-5. screwdriver-plugin-platforms
-
-To override a config value specify it in the config passed to screwdriver-api
-```js
-const API = require('screwdriver-api');
-const Datastore = require('screwdriver-datastore-dynamodb');
-
-const server = new API({
-    datastore: new Datastore(),
-    'screwdriver-plugin-login': {
-        login1: 'value1',
-        login2: 'value2'
-        datastore: new Datastore()
-    }
-    port: 8666
-}, (error, server) {
-    if (error) {
-        return console.error(error);
-    }
-    console.log('Server running at ', server.info.uri);
-});
+## Configuration
+
+Screwdriver already [defaults most configuration](config/default.yaml), but you can override any of it by either be a `local.yaml` or environment variables.
```

Comment:
but you can override **defaults using a** `local.yaml` or environment variables.